LottoDiary/LottoDiary/Sources/Present/Calendar/ViewController/AddLottoViewController.swift

```diff
@@ -0,0 +1,413 @@
+    }()
+    
+    init(lotto: Lotto, onChange: @escaping (Lotto)->Void) {
+        self.lotto = lotto
+        self.onChange = onChange
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .designSystem(.backgroundBlack)
+        purchaseTextField.delegate = self
+        winningTextField.delegate = self
+        setUI()
+    }
+    
+    
+    
+    func setUI() {
+        [typeLabel, lottoSegmentedControl, purchaseTextFieldView, winningTextFieldView, warningPurchaseLabel, warningWinningLabel, okButton, cancelButton]
+            .forEach{ self.view.addSubview($0) }
+        
+        typeLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(15)
+            make.top.equalTo(100)
+        }
+    
+        
+        lottoSegmentedControl.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(15)
+            make.top.equalTo(typeLabel.snp.bottom).offset(20)
+            make.height.equalTo(45)
+        }
+
+        purchaseTextFieldView.snp.makeConstraints { make in
+            make.top.equalTo(lottoSegmentedControl.snp.bottom).offset(30)
+            make.leading.trailing.equalToSuperview().inset(30)
+            make.height.equalTo(80)
+        }
+        
+        purchaseLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(10)
+            make.top.equalToSuperview().offset(10)
+        }
+        
+        purchaseTextField.attributedPlaceholder = NSAttributedString(string: "구매금액을 입력해주세요", attributes: [NSAttributedString.Key.foregroundColor : UIColor.designSystem(.gray63626B)!])
+        
+        
+        purchaseTextField.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(10)
+            make.top.equalTo(purchaseLabel.snp.bottom).offset(5)
+            make.height.equalTo(30)
+            make.width.equalToSuperview()
+        }
+        
+        warningPurchaseLabel.snp.makeConstraints { make in
+            make.top.equalTo(purchaseTextField.snp.bottom).offset(5)
+            make.leading.equalToSuperview().offset(40)
+        }
+        
+        winningTextFieldView.snp.makeConstraints { make in
+            make.top.equalTo(purchaseTextFieldView.snp.bottom).offset(20)
+            make.leading.trailing.equalToSuperview().inset(30)
+            make.height.equalTo(80)
+        }
+        
+        winningLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(10)
+            make.top.equalToSuperview().offset(10)
+        }
+        
+        
+        winningTextField.attributedPlaceholder = NSAttributedString(string: "당첨금액을 입력해주세요", attributes: [NSAttributedString.Key.foregroundColor : UIColor.designSystem(.gray63626B)!])
```

Comment:
```suggestion
        let placeholer = NSAttributedString(string: "당첨금액을 입력해주세요", attributes: [.font : UIFont.gmarksans(weight: .medium, size: ._20), .foregroundColor : UIColor.designSystem(.gray63626B)!])
        winningTextField.attributedPlaceholder = placeholer
```
이런 식으로 폰트까지 지정하면 어떨까 생각해봤습니다. 그리고 placeholder을 똑같은 설정으로 2개를 해야 하니 더 편리하게 아래 사진처럼 customTextField를 만들어 보는 것이 어떨까 생각해봤습니당. 

![스크린샷 2022-12-08 오후 10 52 39](https://user-images.githubusercontent.com/85678496/206463451-557576c4-e89c-4b76-a0dc-73a30e39dc81.png)